concepts/55 Vue Components/20 Component Configuration Syntax/42 Call Methods.md

```diff
@@ -37,9 +37,29 @@ To call UI component methods, you need the UI component instance. Define its key
     };
     </script>
 
+    <!-- tab: App.vue (Composition API) -->
+    <template>
+        <div>
+            <DxTextBox :ref="textBoxRefKey" />
+            <DxButton text="Focus TextBox" @click="focusTextBox" />
+        </div>
+    </template>
+
+    <script setup>
+    import DxButton from 'devextreme-vue/button';
+    import DxTextBox from 'devextreme-vue/text-box';
+
+    import { ref } from 'vue';
+
+    const textBoxRef = ref(null);
+    const focusTextBox = () => {  
+        textBoxRef.value?.instance.focus();
+    };
+    </script>
+
```

Comment:
@GoodDayForSurf Please take a look at this specific part!

Reply:
<DxTextBox :ref="textBoxRefKey" /> is wrong.
Right is: <DxTextBox ref="textBoxRef" />

Reply:
`textBoxRef.value?.instance.focus()` -> `textBoxRef.value.instance.focus()`